types/ltx/package.json

```diff
@@ -0,0 +1,11 @@
+{
+    "private": true,
+    "exports": {
+        ".": {
+            "types": {
+                "import": "./src/ltx.d.ts",
```

Comment:
Both of these entrypoints will be cjs format, afaik, which... works, I guess? I just don't get why.

Reply:
I thought, the first will be mjs. But if you say so, I'll remove the package.json.

Reply:
It'll be the file looked up by an `import`, yes, but it'll only be a `mjs` format file if the file itself has a `.d.mts` extension.

Reply:
It's definitely correct to remove it though - generally speaking, DT packages only need an export map if the source package has one, and it seems like ltx [does not](https://github.com/xmppjs/ltx/blob/main/package.json).

Reply:
Ack, but it does use `"type": "module",` and `"type": "commonjs",` for the `lib` folder - afaik, we'd need to duplicate that - add `"type": "module",` to this package json and add a package json in the lib folder with `"type": "commonjs"`, and we can make this present the same formats. I... do not know if the DT automation is up to the task of allowing a nested `package.json`.

Reply:
Maybe it just works if it's listed in `OTHER_FILES.txt`? I dunno.

Reply:
Let's try it out.

Reply:
OK, the build didn't fail but no idea whether it will be published correctly.